examples/quickstart/local_channel_docs.cpp

```diff
@@ -0,0 +1,63 @@
+//  Copyright (c) 2016 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// This example is meant for inclusion in the documentation.
+
+#include <hpx/hpx_main.hpp>
+#include <hpx/include/apply.hpp>
+#include <hpx/include/iostreams.hpp>
+#include <hpx/include/lcos.hpp>
+
+#include <numeric>
+#include <string>
+#include <vector>
+
+///////////////////////////////////////////////////////////////////////////////
+void minimal_channel()
+{
+    //[local_channel_minimal
+    hpx::lcos::local::channel<int> c;
+    hpx::future<int> f = c.get();
+    HPX_ASSERT(f.is_ready() == false);
```

Comment:
@tiagofgon do the assertions confuse more than they help?

Reply:
If anything, I'd write this as `HPX_ASSERT(!f.is_ready());`, but a comment might help.

Reply:
I think it help, it's understandable. With the next examples become clear what's pretends.

Reply:
Autopilot... I update the assertions.